lib/projectPreprocessor.js

```diff
@@ -272,11 +272,11 @@ class ProjectPreprocessor {
 			return false; // ignore this project
 		}
 
-		if (project.specVersion !== "0.1" && project.specVersion !== "1.0") {
+		if (project.specVersion !== "0.1" && project.specVersion !== "1.0" && project.specVersion !== "1.1") {
 			throw new Error(
 				`Unsupported specification version ${project.specVersion} defined for project ` +
-				`${project.id}. ` +
-				`See https://github.com/SAP/ui5-project/blob/master/docs/Configuration.md#specification-versions`);
```

Comment:
Maybe also adopt the old Configuration.md file to point to the #specifiction-version anchor? Currently it just points to the Configuration page.

Reply:
Will do 👍